File: Vanilla/sources/zdoc/media/lua/shared/Library/StorySoundEvent.lua
---@class StorySoundEvent : zombie.radio.StorySounds.StorySoundEvent
---@field protected name String
---@field protected eventSounds ArrayList|Unknown
StorySoundEvent = {}

---@public
---@param arg0 String
---@return void
function StorySoundEvent:setName(arg0) end

---@public
---@return ArrayList|Unknown
function StorySoundEvent:getEventSounds() end

---@public
---@return String
function StorySoundEvent:getName() end

---@public
---@param arg0 ArrayList|Unknown
---@return void
function StorySoundEvent:setEventSounds(arg0) end

File: Vanilla/sources/zomboid/zombie/radio/StorySounds/StorySoundEvent.java
package zombie.radio.StorySounds;

import java.util.ArrayList;

public final class StorySoundEvent {
   protected String name;
   protected ArrayList eventSounds;

   public StorySoundEvent() {
      this("Unnamed");
   }

   public StorySoundEvent(String var1) {
      this.eventSounds = new ArrayList();
      this.name = var1;
   }

   public String getName() {
      return this.name;
   }

   public void setName(String var1) {
      this.name = var1;
   }

   public ArrayList getEventSounds() {
      return this.eventSounds;
   }

   public void setEventSounds(ArrayList var1) {
      this.eventSounds = var1;
   }
}
